cmd/rm-main.go

```diff
@@ -209,14 +214,14 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, older int) erro
 			return exitStatus(globalErrorExitStatus)
 		}
 
-		if older > 0 {
-			// Check whether object is created older than given time.
-			now := time.Now().UTC()
-			timeDiff := now.Sub(content.Time)
-			if timeDiff < (time.Duration(older) * Day) {
-				// time difference of info.Time with current time is less than older time.
-				continue
-			}
+		// Skip objects older than --older-than parameter if specified
+		if olderThan > 0 && isOlder(content, olderThan) {
+			continue
+		}
+
+		// Skip objects newer than --newer-than parameter if specified
+		if newerThan > 0 && isNewer(content, newerThan) {
```

Comment:
I see we could use `--older-than` only, `newer-than` only or `older-than` and `newer-than` together.  We would need to add proper examples `CustomHelpTemplate` in for those combinations for applicable commands.

Similarly I couldn't able to understand `older-than` and `newer-than` combination use case.